WIP Editing @alimanfoo 2019-09-25

alimanfoo opened the pull request:
This PR has editing of the main text, including:

* Add citations and tighten up the introduction
* Discussion of CNV markers of MFO-mediated pyrethroid resistance
* Discussion of resistance surveillance more broadly
* Discussion of gene flow
* Tighten up discussion of gene flow and add citations
* Round off discussion
* Edit results text on CNV PCA
* Add dsx citations
* Add remaining discussion citations

alimanfoo commented:
Hi @cclarkson, this is still WIP but comments very welcome.

alimanfoo commented:
Trying to find a balance between talking about potential for surveillance, but not going into too much depth (given that this is a resource paper and not a surveillance paper) and also avoiding getting too political or opinionated. Tricky. Suggestions very welcome.

cclarkson commented on main.tex:
```diff
@@ -489,106 +489,69 @@ \section*{Discussion}
+It would therefore be valuable to survey mosquito populations and determine the prevalence of different pyrethroid resistance mechanisms, both before and after any change in vector control strategy.
 %
-In Phase 2 this subset consists of 44,076,901 SNPs.
+There are currently no genetic assays for surveillance of pyrethroid metabolic resistance in \textit{An. gambiae} or \textit{An. coluzzii}, but our data resource includes CNVs at two MFO loci (\textit{Cyp6p/aa}, \textit{Cyp9k1}) and one GST locus (\textit{Gste}) which could serve as molecular markers of metabolic resistance mechanisms.
```
We could perhaps push more strongly here  - our data includes these loci which have been proven to be involved in pyrethroid metabolism in mosquitoes.

cclarkson commented:
Looks good so far :+1:

alimanfoo commented:
For visibility, here are the CNV alleles data used to discuss gene flow. Here "S" means the allele is in gambiae, "S*" means it's in more than one gambiae population, "M" means it's in coluzzii, "M*" means it's in more than one coluzzii population.

Cyp6aa/p
S - Dup1a - UGS (65/112)
M - Dup1b - BMF (6/75)
S* - Dup2 - BFS (1/92), GNS (1/40)
S - Dup3 - CMS (1/297)
M - Dup4 - BFM (3/75)
S - Dup5 - UGS (5/112)
S - Dup6 - CMS (1/297)
M* - Dup7 - BFM (33/75), CIM (23/71), GHM (3/55), GNM (3/4)
S* - Dup8 - BFS (3/92), GNS (1/40)
S - Dup9 - UGS (1/112)
M* - Dup10 - BFM (37/75), GHM (3/55)
M* - Dup11 - CIM (29/71), GHM (3/55)
S - Dup12 - UGS (1/112)
M - Dup13 - CIM (1/71)
M* - Dup14 - BFM (2/75), CIM (33/71)
M* - Dup15 - BFM (1/75), CIM (28/71)

S*/S: 2/8; M*/M: 5/8


Gste
M - Dup1 = BFM (16/75)
S - Dup2 = UGS (15/112)
S - Dup3 = UGS (22/112)
S - Dup4 = CMS (4/297)
S* - Dup5 = CMS (7/297), GAS (9/69)
S - Dup6 = UGS (1/112)
M* - Dup7 = BFM (3/75), GHM (6/55)
S - Dup8 = UGS (4/112)
M - Dup9 = AOM (11/78)
S - Dup10 = CMS (2/297)
Dup11 = KE (23/48)

S*/S: 1/7; M*/M: 1/3

Cyp9k1
S - Dup1 = CMS (2/297)
S - Dup2 = CMS (2/297)
FW - Dup3 = GW (2/91)
S* - Dup4 = BFS (19/92), CMS (4/297), GHS (2/12), GNS (6/40)
S - Dup5 = UGS (3/112)
M - Dup6 = AOM (12/78)
S* - Dup7 = BFS (9/92), CMS (1/297)
S - Dup8 = UGS (32/112)
S - Dup9 = CMS (5/297)
M* - Dup10 = BFM (1/75), GHM (3/55)
S* - Dup11 = BFS (59/92), CMS (4/297), GHS (3/12), GNS (24/40)
FW* - Dup12 = GM (38/65), GW (8/91)
S* - Dup13 = BFS (2/92), GNS (1/40)
S - Dup14 = BFS (2/92)
S* - Dup15 = BFS (45/92), GHS (2/12), GNS (12/40)
S - Dup16 = UGS (3/112)

S*/S: 5/12; M*/M: 1/2.

Combined S*/S: (5+1+2)/(12+7+8) = 8/27.
Combined M*/M: (1+1+5)/(2+3+8) = 7/13.

alimanfoo commented:
Hi @cclarkson, I've done some work on the intro here, adding citations and tightening up the text. But I'm behind on my other tasks to finish off the discussion, and will need some time today (Wednesday) to finish those off. Fortunately I have no meetings so can devote the whole day to it, suggest we aim to lock and send by end of Wednesday if that's OK with you.

cclarkson commented on main.tex:
```diff
@@ -257,24 +255,24 @@ \subsection*{Species assignment}
 In particular, all individuals from two sampling locations (Kenya, Guinea-Bissau) carried a mixture of \textit{An. gambiae} and \textit{An. coluzzii} alleles, creating uncertainty regarding the appropriate species assignment \cite{Ag1000gConsortium2017}. 
 %
-Applying the same analysis to the new samples in Ag1000G Phase 2, we found that mosquitoes from The Gambia also carried a mixture of alleles from both species, in similar proportions to mosquitoes from Guinea-Bissau (Figure S1).
+Applying the same analysis to the new samples in Ag1000G Phase 2, we found that mosquitoes from The Gambia also carried a mixture of alleles from both species, in similar proportions to mosquitoes from Guinea-Bissau (Figure \ref{aim}).
```
If the phases are to be lower case, then there are a still a few upper case that need sorting.

alimanfoo replied:
Wasn't sure what it should be, opted for lower case at random. I can do a pass to make uniform.

cclarkson replied:
I had made them upper case when specifically referring to "Ag1000G Phase 1/2", but as @hardingnj  once famously said "...it's not a hill I am prepared to die on." (-; 

- I don't think it matters as long as they're the same.

cclarkson commented on main.tex:
```diff
@@ -506,114 +488,128 @@ \section*{Discussion}
+There are currently no genetic assays for surveillance of pyrethroid metabolic resistance in \textit{An. gambiae} or \textit{An. coluzzii}, but our data resource includes CNVs at two CYP loci (\textit{Cyp6p/aa}, \textit{Cyp9k1}) which could serve as molecular markers of CYP-mediated metabolic resistance.
 %
-Whole genome phasing was performed using these biallelic variants, estimating chromosomal haplotypes for each sample, a data resource useful for many population genetic analyses including scans for selection.
+Glutathione S-transferase enzymes have also been implicated in metabolic resistance to pyrethroids, but are not inhibited by PBO, and we found CNVs at the \textit{Gste} locus which could serve as molecular markers of this alternative resistance mechanism.  
```
Maybe switch this sentence around a bit - I found the "..but are not" in the middle a bit confusing the first time I read it.

Maybe 

"Glutathione S-transferase enzymes have also been implicated in metabolic resistance to pyrethroids and we found CNVs at the \textit{Gste} locus which could serve as molecular markers of this alternative resistance mechanism, though these enzymes are not inhibited by PBO."

alimanfoo replied:
Thanks, I've addressed.

alimanfoo commented on main.tex:
```diff
@@ -506,134 +488,156 @@ \section*{Discussion}
+Glutathione S-transferase enzymes have also been implicated in metabolic resistance to pyrethroids, but are not inhibited by PBO, and we found CNVs at the \textit{Gste} locus which could serve as molecular markers of this alternative resistance mechanism.  
 %
-Alongside meta-data for each sample containing details on where and when the sample was collected we have also added presence/absence information for known insecticide resistance loci.
+Further work is needed to characterise the resistance phenotype associated with these CNVs, but the allelic heterogeneity, the high population frequencies, and the evidence for positive selection observed in our data, coupled with previous gene expression and functional studies @@REFs, all support a metabolic role in pyrethroid resistance.
```
Hey @cclarkson, could you tell me which papers I should cite here?

cclarkson replied:
\cite{edi2014,  vontas2018} would definitely work here

alimanfoo commented on main.tex:
```diff
@@ -506,134 +488,156 @@ \section*{Discussion}
-\textbf{2.} Regions with both high target-site and high metabolic pyrethroid resistance associated loci frequencies, in these regions many individuals carry markers for both modes of resistance.
-%to vector control
-The outcome of PBO nets is harder to predict in these regions, as the addition of the synergist can only reduce the metabolic resistance mediated fraction of the pyrethroid resistance phenotype.
+Current WHO guidance states that PBO LLINs should only be deployed in regions of ``moderate resistance'', and may provide no additional benefit in regions of ``high resistance'', presumably because high levels of resistance may indicate that multiple resistance mechanisms are present @@REF.
```
I'll look this citation up.

alimanfoo commented on main.tex:
```diff
@@ -506,134 +488,156 @@ \section*{Discussion}
+There are several options for scaling up surveillance of new genetic markers, including both whole genome sequencing (WGS) and targeted (amplicon) sequencing with several choices of sequencing technology platform, as well as various PCR-based assays.
 %
-In order to extract the maximum value from future genomic insecticide resistance monitoring we suggest a two stage approach.
+Assays that target specific genetic loci are attractive in the short term, because of the low cost and infrastructure requirements, and data from Ag1000G have been used successfully to design multiplex assays for the Agena MassArray platform and for Illumina amplicon sequencing @@REFs.
```
@EricRLucas is your sequenom paper citable?

EricRLucas replied:
Yep:
@ARTICLE{Luc19b,
  author = {Lucas, E. R. and Rockett, K. A. and Lynd, A. and Essandoh, J. and
	Grisales, N. and Kemei, B. and Njoroge, H. and Hubbart, C. and Rippon,
	E. J. and Morgan, J. and Van’t Hof, A. E. and Ochomo, E. O. and Kwiatkowski,
	D. P. and Weetman, D. and Donnelly, M. J.},
  title = {A high throughput multi-locus insecticide resistance marker panel
	for tracking resistance emergence and spread in Anopheles gambiae},
  journal = {Scientific Reports},
  year = {2019},
  volume = {9},
  pages = {13335},
  doi = {10.1038/s41598-019-49892-6}
}

alimanfoo replied:
Perfect, thanks!

alimanfoo commented:
Hey @cclarkson, I'm good to go here, except for a few outstanding citations. I need a break now but will come back to those shortly. In the mean time, any comments let me know :-)

cclarkson commented:
Hi @alimanfoo,

I'll give this PR another pass from the beginning and comment/fill in citations, then we can discuss at 5pm if that's still cool (a bit earlier if possible, but if not, 5 is fine)?

alimanfoo commented:
Cool, thanks @cclarkson. I'm working on dsx references now. If you find any references for other parts, probably easiest to add as a comment with a suggestion, then I can merge into this PR.

cclarkson commented on main.tex:
```diff
@@ -88,58 +88,56 @@ \section*{Introduction}
-Also, only SNPs were studied in Ag1000G Phase 1, but other types of genetic variation are known to be important.
+Also, only SNPs were studied in Ag1000G phase 1, but other types of genetic variation are known to be important.
 %
 In particular, copy number variation (CNV) has long been suspected to play a key role in insecticide resistance \cite{schimke1978, devonshire1991, weetman2015}, but no previous attempts to call genome-wide CNVs have been made in these species.
```
I've put spaces in between citations, you haven't - need changing to follow one format.

cclarkson commented:
Hi @alimanfoo, for some reason I cannot 'quickbuild' a pdf with your main.tex - I keep getting the error  `! missing $ inserted` - I can't find the issue.

alimanfoo commented:
Hi @cclarkson, I've merged in master to this branch, which includes the two PRs you had added on CNV PCA methods and reference divergence. I also found the '$' error you were getting, it was in the R reference, that's fixed. If you pull this branch now, should build and be good for a read through.

cclarkson commented on main.tex:
```diff
@@ -394,68 +384,64 @@ \subsection*{Design of Cas9 gene drives}
 
 %%
-Malaria control in Africa depends heavily on mass distribution of long-lasting insecticidal bed-nets (LLINs) impregnated with pyrethroid insecticides \cite{Ranson2016, churcher2016}.
+Malaria control in Africa depends heavily on mass distribution of long-lasting insecticidal bed-nets (LLINs) impregnated with pyrethroid insecticides \cite{Ranson2016, churcher2016} @@REF Bhatt.
```
Hi @alimanfoo - shall we just add the Bhatt here?

cclarkson replied:
Also, lower-case "chromosome" here. Inconsistent.

cclarkson commented on main.tex:
```diff
@@ -283,43 +281,33 @@ \subsection*{Population structure}
-We investigated genetic population structure within the cohort of wild-caught mosquitoes by performing principal components analysis (PCA) (Figure 2).
-%
-To complement PCA, we also computed two measures of genetic differentiation -- average $F_{ST}$ and rates of rare variant sharing -- between all pairs of 16 populations defined by country of origin and species, excluding \textit{An. coluzzii} from Guinea due to small sample size (Figure 1, Table 1).
+We investigated genetic population structure within the cohort of wild-caught mosquitoes by performing two principal components analyses (PCA), the first using biallelic SNPs from euchromatic regions of Chromosome 3 (Figure \ref{fig:pca}), the second using CNVs from the whole genome (Figure \ref{fig:cnv_pca}).
```
Do we not mention the cnv pca again? If not, is there any point in it being there (this is a note to myself - I might find another mention in a bit!)

alimanfoo replied:
No I didn't mention it again, I think it basically corroborates the SNP PCA.

cclarkson commented on main.tex:
```diff
@@ -349,35 +339,35 @@ \subsection*{Genetic diversity}
 %% Summary of main patterns in the diversity data.
 %
-The two Eastern-most populations (Kenya, Mayotte \textit{An. gambiae}) were outliers in all statistics calculated, with lower diversity, a deficit of rare variants relative to neutral expectation, and a higher degree of haplotype sharing within and between individuals.
+The two Eastern-most populations (Kenya, Mayotte) were outliers in all statistics calculated, with lower diversity, a deficit of rare variants relative to neutral expectation, and a higher degree of haplotype sharing within and between individuals.
```
I think the word here is "easternmost"

cclarkson commented on main.tex:
```diff
@@ -394,68 +384,64 @@ \subsection*{Design of Cas9 gene drives}
+Insect sex determination depends on sex-specific splicing of \textit{dsx}, with exon 5 being retained in the female transcript and skipped in the male transcript \cite{Gempe2011}.
 %
-Any sequences required for the proper binding of this splicing machinery to the pre-mRNA will be subject to strong purifying selection, because disruption of \textit{dsx} splicing is highly deleterious in females @@REF. 
+In \textit{Drosophila melanogaster} this differential splicing depends on female-specific regulatory factors which bind to the exon 5 region of the \textit{dsx} pre-mRNA and promote inclusion of exon 5 within the final transcript \cite{Cline1996, Gempe2011}.
```
Capitalised "Chromosome" but not "exon"

alimanfoo replied:
Kyrou use "exon". Editors have told me in the past to capitalise e.g. "Chromosome 2".

cclarkson commented on main.tex:
```diff
@@ -505,135 +487,166 @@ \subsection*{Resistance to pyrethroid insecticides}
+There are currently no genetic assays for surveillance of pyrethroid metabolic resistance in \textit{An. gambiae} or \textit{An. coluzzii}, but our data resource includes CNVs at two CYP loci (\textit{Cyp6p/aa}, \textit{Cyp9k1}) which could serve as molecular markers of CYP-mediated metabolic resistance.
 %
-Whole genome phasing was performed using these biallelic variants, estimating chromosomal haplotypes for each sample, a data resource useful for many population genetic analyses including scans for selection.
+Glutathione S-transferase enzymes have also been implicated in metabolic resistance to pyrethroids @@REFs, and we found CNVs at the \textit{Gste} locus which could serve as molecular markers of this alternative resistance mechanism, which is not inhibited by PBO.  
```
It's in funestus but this paper is great.

@article{riveron2014single,
  title={A single mutation in the GSTe2 gene allows tracking of metabolically based insecticide resistance in a major malaria vector},
  author={Riveron, Jacob M and Yunta, Cristina and Ibrahim, Sulaiman S and Djouaka, Rousseau and Irving, Helen and Menze, Benjamin D and Ismail, Hanafy M and Hemingway, Janet and Ranson, Hilary and Albert, Armando and others},
  journal={Genome biology},
  volume={15},
  number={2},
  pages={R27},
  year={2014},
  publisher={BioMed Central}
}

cclarkson commented on main.tex:
```diff
@@ -742,11 +756,12 @@ \subsection*{1.2 Colony populations}
+\subsection*{Whole genome sequencing}
 
 %
 Sequencing was performed on the Illumina HiSeq 2000 platform at the Wellcome Trust Sanger Institute.
```
Now "Wellcome Sanger Institute"

cclarkson commented on main.tex:
```diff
@@ -779,27 +795,52 @@ \subsection*{2.1 Genome accessibility}
+\subsection*{Reference divergence}
+
+%%
+The genomes of \textit{An. coluzzii} and \textit{An. gambiae} samples were similarly diverged from the reference genome used for alignment, AgamP4 \cite{Holt2002}, indicating that unbiased comparisons can be made between the two species based on our results (Fig. \ref{refdiff}a).
```
figure is broken, "\ref{NAME}" needs updating.

cclarkson commented on main.tex:
```diff
@@ -852,55 +893,41 @@ \subsection*{2.5 SNP validation}
+\section*{The \textit{Anopheles gambiae} 1000 Genomes Consortium}
 
 %
 \textbf{Data analysis group}: Alistair Miles (project lead), Nicholas J. Harding, Giordano Bott\'{a}, Chris S. Clarkson, Tiago Ant\~{a}o, Krzysztof Kozak, Daniel R. Schrider, Andrew D. Kern, Seth Redmond, Igor Sharakhov, Richard D. Pearson, Christina Bergey, Michael C. Fontaine, Martin J. Donnelly, Mara K. N. Lawniczak and Dominic P. Kwiatkowski (chair).
```
This needs C.J. Battey adding

cclarkson commented:
Hi @alimanfoo, it looks great. There are only a few comments/broken links to fix (see above).

Apologies it took me until now to get through it, Helen got held up at work. Wasn't sure if I could make the changes on your branch without breaking git. Happy to help fixing it if I can, just let me know.

alimanfoo commented:
Thanks a load @cclarkson, I'm working through the last bits and pieces now, nearly there :-)

alimanfoo commented:
Hi @cclarkson, I've addressed your comments and also done some edits to the list of authors to reflect contributions to this project phase. 

I noticed that we don't have authors for Ghana or Cote d'Ivoire, do you have those already? If not we can get them off Martin and fill them in before sending to the consortium.
